model/config.go

```diff
@@ -176,6 +176,8 @@ const (
 
 	ExperimentalSettingsDefaultLinkMetadataTimeoutMilliseconds = 5000
 
+	ExperimentalSettingsDefaultInteractiveDialogTriggerTimeoutMilliseconds = 5000
```

Comment:
I'm thinking we should also remove the currently used constant for this feature. The code using that constant needs to be adjusted to use this new config setting as well.

Reply:
Sure. I would like to finish it. I've reopened the pr which I made for the web app.

So I only need to remove this constant ? is there anything else that needs to be done ?